recipes/openfst/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: openfst
+  version: "1.6.1"
```

Comment:
Please use a jinja variable for the version to avoid problems with different number in the future.